--- pgocaml-1.0/pGOCaml.mli~	2011-07-18 13:44:46.752487738 +0200
+++ pgocaml-1.0/pGOCaml.mli	2011-07-18 13:48:17.948487688 +0200
@@ -1,6 +1,8 @@
 (* PG'OCaml is a set of OCaml bindings for the PostgreSQL database.
  * $Id$
  *)
+open CalendarLib
+open Calendar
 
 type 'a t				(** Database handle. *)
 
--- pgocaml-1.0/pGOCaml.ml~	2007-10-28 21:05:44.000000000 +0100
+++ pgocaml-1.0/pGOCaml.ml	2011-07-18 13:49:33.651487672 +0200
@@ -4,6 +4,8 @@
 
 open Printf
 open ExtString
+open CalendarLib
+open Calendar
 
 type 'a t = {
   ichan : in_channel;			(* In_channel wrapping socket. *)
